docs/datascience/overview.md

```diff
@@ -0,0 +1,29 @@
+---
+Order: 1
+Area: datascience
+TOCTitle: Overview
+ContentId: 17345073-cb40-448c-a312-28982900f132
+PageTitle: Doing Data Science in Visual Studio Code
+DateApproved: 5/27/2021
+MetaDescription: Doing Data Science in Visual Studio Code.
+---
+
+# Data Science in Visual Studio Code
+
+You can do all of your data science work within VS Code! Use Jupyter Notebooks and the [Interactive Window](/docs/python/jupyter-support-py.md) to start analyzing and visualizing your data in minutes! Power your Python coding experience with Intellisense support and build, train, and deploy machine learning models to the cloud or the edge with Azure Machine Learning service.
+
+![Preview of Jupyter Notebooks in VS Code](images/overview/jupyter-notebook-preview.png)
+
+## Extensions
+
+The VS Code Marketplace offers a family of extensions that provide a first-class data science experience for Python data science. In order to get started with Python, Microsoft strongly recommends the following extensions:
+
+<div class="marketplace-extensions-datascience-python"></div>
```

Comment:
@tanhakabir I'm most likely wrong but figured I'd ask just in case hehe 
is it missing an extension list here by any chance?

Reply:
Oh yeah sorry this list is on a different repo, the vscode-website repo but you can see the list I have in the screenshots above

Reply:
Cool, thank you!! 😊